.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+	"printWidth": 80,
+	"tabWidth": 4,
```

Comment:
I think we should update the tabWidth to 2. agree?
```

Reply:
I agree with you, Actually, I prefer to remove tabWidth which means the default value and it is 2. But I didn't want to replace all other previous configs which have been set(I was planning to make it in 2 or more steps to take some feedback about it and engage others to tell their preferences), The only  needed option which is not in the default config of prettier is:
```
{
  "endOfLine": "auto"
}
```
But if we want to be precautious about what will be the default in the future, I prefer to use this:
```
{
  "printWidth": 80,
  "semi": true,
  "trailingComma": "all",
  "endOfLine": "auto"
}
```
 About `printWidth`, I prefer the `80` because it is an already accepted convention through almost every other projects/languages.
May I have your opinion on this?

Thanks for this much detailed review🙏🌹

Reply:
I agree with you to remove the ```tabWidth```.